tests/conformance/state/state.go

```diff
@@ -1016,6 +1026,174 @@ func ConformanceTests(t *testing.T, props map[string]string, statestore state.St
 				return res.Data == nil
 			}, time.Second*3, 200*time.Millisecond, "expected object to have been deleted in time")
 		})
+
+		t.Run("no TTL should not return any expire time", func(t *testing.T) {
+			err := statestore.Set(context.Background(), &state.SetRequest{
+				Key:      key + "-no-ttl",
+				Value:    "⏱️",
+				Metadata: map[string]string{},
+			})
+			require.NoError(t, err)
+
+			// Request immediately
+			res, err := statestore.Get(context.Background(), &state.GetRequest{Key: key + "-no-ttl"})
+			require.NoError(t, err)
+			assertEquals(t, "⏱️", res)
+
+			assert.NotContains(t, res.Metadata, "ttlExpireTime")
+		})
+
+		t.Run("ttlExpireTime", func(t *testing.T) {
+			if !config.HasOperation("transaction") {
+				t.Skip("state store does not support transactions")
```

Comment:
```suggestion
				// This test is only for state stores that support transactions
				return
```

Reply:
Let's not pollute logs for things that are never going to change, it can cause confusion